lib/rubocop/cop/style/random_with_offset.rb

```diff
@@ -36,15 +36,15 @@ class RandomWithOffset < Base
             (send
               {nil? (const {nil? cbase} :Random) (const {nil? cbase} :Kernel)}
               :rand
-              {int irange erange}))
+              {int (irange (int _) (int _)) (erange (int _) (int _))}))
```

Comment:
No biggie, but `(int _)` => `int` is simpler

Reply:
Thanks, updated!